File: TapIn/app/src/main/java/com/tapin/tapin/model/BusinessInfo.java
package com.tapin.tapin.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

/**
 * Created by User on 03/10/2017.
 */

public class BusinessInfo implements Cloneable, Serializable {

    @SerializedName("status")
    public int status;

    @SerializedName("data")
    public List<Business> listBusinesses;
}

File: TapIn/app/src/main/java/com/tapin/tapin/model/BusinessInfo.kt
package com.tapin.tapin.model

import com.google.gson.annotations.SerializedName
import com.tapin.tapin.model.resturants.Business

import java.io.Serializable

/**
 * Created by User on 03/10/2017.
 */

class BusinessInfo : Cloneable, Serializable {

    @SerializedName("status")
    var status: Int = 0

    @SerializedName("data")
    var listBusinesses: List<Business>? = null
}
